rholang/src/main/scala/coop/rchain/rholang/interpreter/matcher/StreamT.scala

```diff
@@ -149,6 +149,11 @@ trait StreamTInstances2 {
   private[matcher] type of[F[_], G[_]] = { type l[A] = F[G[A]] }
   private[matcher] type StreamTC[M[_]] = { type l[A] = StreamT[M, A] }
 
+  implicit def catsDataMonadErrorMonadForStreamT[F[_]](
+      implicit F0: Monad[F]
+  ): MonadError[StreamT[F, ?], Unit] =
+    new StreamTMonadErrorMonad[F] { implicit val F = F0 }
```

Comment:
Just because it was possible to avoid a def that would be re-evaluated upon call. The benefit is probably cancelled by the confusion this caused... OTOH changing that now would also not pay off :)